votingpool/db.go

```diff
@@ -116,17 +185,18 @@ func putSeriesRow(tx walletdb.Tx, votingPoolID []byte, ID uint32, row *dbSeriesR
 	bucket, err := tx.RootBucket().CreateBucketIfNotExists(votingPoolID)
 	if err != nil {
 		str := fmt.Sprintf("cannot create bucket %v", votingPoolID)
-		return managerError(waddrmgr.ErrDatabase, str, err)
+		return newError(ErrDatabase, str, err)
 	}
+	bucket = bucket.Bucket(seriesBucketName)
 	serialized, err := serializeSeriesRow(row)
 	if err != nil {
 		str := fmt.Sprintf("cannot serialize series %v", row)
-		return managerError(waddrmgr.ErrSeriesStorage, str, err)
+		return newError(ErrSeriesStorage, str, err)
```

Comment:
I noticed that some of the `ErrSeriesStorage` errors have an underlying error and some don't.  This seems a bit inconsistent with the rest of the code where `ErrDatabase` always has an underlying error and that others don't.

I suppose this is fine as long as it's documented, but it is a departure from the rest of the code.